SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -600,6 +603,35 @@ void USpatialNetDriver::QueryGSMToLoadMap()
 	GlobalStateManager->QueryGSM(QueryDelegate);
 }
 
+void USpatialNetDriver::OnActorSpawned(AActor* Actor)
+{
+	if (!Actor->GetIsReplicated())
+	{
+		return;
+	}
+
+	if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton | SPATIALCLASS_NotSpatialType))
+	{
+		return;
+	}
+
+	if (Actor->GetLocalRole() != ROLE_Authority)
+	{
+		return;
+	}
+
+	if (USpatialStatics::IsActorGroupOwnerForActor(Actor))
```

Comment:
If you're not using offloading, are is the default Actor group or all Actors just the UnrealWorker one?

Reply:
Basically, yes, I think it's called the default actor group and the UnrealWorker is the owner of all of it.